src/common/transactions/requests.ts

```diff
@@ -0,0 +1,73 @@
+import { getAppPrivateKey, Wallet } from '@stacks/wallet-sdk';
+import { TransactionPayload } from '@stacks/connect';
+import { decodeToken, TokenVerifier } from 'jsontokens';
+import { getPublicKeyFromPrivate } from '@stacks/encryption';
+import { getAddressFromPrivateKey, TransactionVersion } from '@stacks/transactions';
+
+function getTransactionVersionFromRequest(tx: TransactionPayload) {
```

Comment:
Why do we sometimes spell out `Transaction` and other times `Tx`?

Reply:
sloppy, idk which is better, this line is already long so it was a lot to write more letters lol